src/signals/incident-management/containers/IncidentDetail/components/Detail/index.js

```diff
@@ -1,68 +1,90 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Paragraph, themeColor, themeSpacing } from '@datapunt/asc-ui';
 
-import { incidentType, dataListType, attachmentsType } from 'shared/types';
+import { incidentType, attachmentsType } from 'shared/types';
 
 import { string2date, string2time } from 'shared/services/string-parser/string-parser';
 
-import './style.scss';
-
-import Highlight from '../Highlight';
 import Location from './components/Location';
 import Attachments from './components/Attachments';
 import ExtraProperties from './components/ExtraProperties';
 
-class Detail extends React.Component { // eslint-disable-line react/prefer-stateless-function
-  render() {
-    const {
-      incident, attachments, stadsdeelList, onShowLocation, onEditLocation, onShowAttachment,
-    } = this.props;
-
-    return (
-      <article className="detail">
-        <div className="detail__text" data-testid="detail-title">
-          {incident.text}
-        </div>
-
-        <dl>
-          <dt className="detail__definition">Overlast</dt>
-          <dd className="detail__value">
-            {string2date(incident.incident_date_start)} {string2time(incident.incident_date_start)}&nbsp;
-          </dd>
-
-          <Highlight
-            subscribeTo={incident.location}
-          >
-            <Location
-              incident={incident}
-              stadsdeelList={stadsdeelList}
-              onShowLocation={onShowLocation}
-              onEditLocation={onEditLocation}
-            />
-          </Highlight>
-
-          <Attachments
-            attachments={attachments}
-            onShowAttachment={onShowAttachment}
-          />
-
-          {incident.extra_properties && <ExtraProperties items={incident.extra_properties} />}
-
-          <dt className="detail__definition" data-testid="detail-email-definition">E-mail melder</dt>
-          <dd className="detail__value" data-testid="detail-email-value">{incident.reporter.email}</dd>
-
-          <dt className="detail__definition" data-testid="detail-phone-definition">Telefoon melder</dt>
-          <dd className="detail__value" data-testid="detail-phone-value">{incident.reporter.phone}</dd>
-        </dl>
-      </article>
-    );
+const DemiParagraph = styled(Paragraph)`
+  font-family: Avenir Next LT W01 Demi;
```

Comment:
I think we can enforce the use of this font automatically just by using `font-weight: 700`

Reply:
How would we do that?

Reply:
This works if you use asc-assets fonts:
- in global.css (and overall where an other name is used): 
```
body {
  font-family: 'Avenir Next', 'Helvetica Neue', Helvetica, Arial, sans-serif;
}
```
- in webpack
```
    new CopyWebpackPlugin([
      {
        from: './node_modules/@datapunt/asc-assets/static/fonts',
        to: 'fonts',
      },
    ]),
```
- in index.html
```
    <!-- This is the tracking code for the fastfonts.net typography -->
    <script src="/fonts/mtiFontTrackingCode.min.js"></script>
    <link href="/fonts/fonts.css" rel="stylesheet" />

```
- in the rest of the application use `font-weight: 700` for demi and , `font-weight: 500`  for medium.

see './node_modules/@datapunt/asc-assets/static/fonts' for the font face options. Only the design system fonts are defined there

Reply:
Ah, maybe for another pull request